chris/zigzag-conversion.go

```diff
@@ -0,0 +1,53 @@
+package chris
+
+import (
+	"strings"
+)
+
+func convert(s string, numRows int) string {
+
+	if len(s) == 1 || numRows == 1 {
```

Comment:
I wonder how much faster mine would have been if I'd had this detection.

Reply:
Not sure it saves much time, just the odd test case that only has 1 item.

Reply:
Ya, that's what I mean. Those test cases would be complete faster, improving your speed.